gradle-plugin/README.md

```diff
@@ -0,0 +1,5 @@
+This directory contains a Gradle plugin for generating specific metrics from
```

Comment:
The plugin does a bit more now: it also generates docs :) Maybe we should be more generic as suggested in the description in the buildconfig file?